toolbox-reference_5b7bdc1d0428631d7a8a2fcc.md

```diff
@@ -236,33 +234,64 @@ The `sem-service` script is a utility for starting, stopping and getting the
 status of background services. Started services will listen on 0.0.0.0 and
 their default port. The 0.0.0.0 IP address includes all available network
 interfaces. Essentially, you will be using services as if they were natively
-installed in the OS.
+installed in the Operating System.
 
 ### Command Line Parameters
 
 The general form of a `sem-service` command is as follows:
 
-    sem-service start [mysql | postgres | redis | memcached]
+    sem-service start [mysql | postgres | redis | memcached | mongodb | elasticsearch] [version]
+
+Therefore, each `sem-service` command requires at least two parameters: the
+first one is the task you want to perform and the second parameter is the name
+of the service that will be used for the task. The third parameter is optional
+and is the version of the service that you want to start. If no `version` value
+is given, a default value will be used according to the following list:
+
+* mysql: The default value is `5.6`
+* postgres: The default value is `9.6`
+* redis: The default value is `4`
+* memcached: The default value is `1.5`
+* mongodb: The default value is `4.1`
+* elasticsearch: The default value is `6.5.1`
+
+`sem-service` pulls images from Docker Hub and supports all versions that are
+available in Docker Hub. You can find the list of available versions at the
+following URLs:
+
+* ElasticSearch: https://hub.docker.com/_/elasticsearch/
```

Comment:
Please check later if URLs resolve to clickable link on Help Scout. If not please create links in Markdown.